pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+This is just a checklist to support you. You can remove it safely.
+
+[ ] If you are new to scapy: I have checked  https://github.com/secdev/scapy/blob/master/CONTRIBUTING.md (esp. section submitting-pull-requests)
+[ ] I squashed commits belonging together
+[ ] I added some tests for the specific code I worked on / adapted exiting tests
+[ ] I executed the regression tests for python2 + 3 (cd test && ./run_tests_py2, cd test && ./run_tests_py3)
```

Comment:
[ ] add unit tests or explain why their are not relevant.

Reply:
replaced the list item 'I added some tests for the specific code I worked on / adapted ' with your text cause it is more 'to the point'...